src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -135,6 +135,7 @@
   --#{$menu-toggle}__button--PaddingInlineStart: var(--pf-t--global--spacer--sm);
   --#{$menu-toggle}__button--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
   --#{$menu-toggle}__button--MinWidth: calc(var(--#{$menu-toggle}--FontSize) * var(--#{$menu-toggle}--LineHeight) + var(--#{$menu-toggle}--PaddingBlockStart) + var(--#{$menu-toggle}--PaddingBlockEnd));
+  --#{$menu-toggle}__button--m-text--PaddingInlineStart: var(--#{$pf-global}--spacer--xs);
```

```suggestion
  --#{$menu-toggle}__button--m-text--PaddingInlineStart: var(--pf-t--global--spacer--xs);
```

To match rules above?  It looks like the `#{$pf-global}` isn't used here in V6, though core devs can confirm if this is needed

---

Oh you are correct, will update.